catalyst-gateway/bin/src/service/api/health/started_get.rs

```diff
@@ -10,6 +10,10 @@ pub(crate) enum Responses {
     /// Service is Started and can serve requests.
     #[oai(status = 204)]
     NoContent,
+    /// Service is not ready, do not send other requests.
+    #[oai(status = 503)]
+    #[allow(dead_code)]
```

Comment:
Not sure why this is "dead_code"?

This endpoint should have a global state variable that defaults to false.
When cat-gateway is finished "starting", ie it has started all its internal services, and is ready to serve requests, it should get set to true (by a function call).
Then endpoint then should check,  if started, return NoContent else return ServiceUnavailable